css/index.less

```diff
@@ -0,0 +1,160 @@
+/* colors used throughout the page */
+@dark-brown: hsl(31, 49%, 10%);
+@light-brown: lighten(@dark-brown, 50%);
+@near-black: hsl(220, 6%, 10%);
+@red: hsl(0, 100%, 50%);
+@lighter-brown: lighten(@light-brown, 15%);
+/* set box-sizing, margin, padding, and color for html and body */
+html, body {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  margin:0;
+  padding:0;
+  color: @dark-brown;
+}
+/* set background image for the body */
+body {
+  background-image: url('../img/shopping-center-iot.jpg');
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  height: 45.938rem;
+  background-position: center bottom;
+  overflow: auto;
+  padding:0;
+  margin: auto;
+  font-family: 'Architects Daughter', cursive;
+  background-repeat: no-repeat;
+}
+/* add fornt to h1 and h3 */
+h1, h3 {
+  font-family: 'Bangers', cursive;
+}
+/* set font size for h1 */
+h1 {
+  font-size: 64px;
+}
+/* reset components */
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
+  -moz-box-sizing: inherit;
+  box-sizing: inherit;
+}
+/* set color, radius, border and opacity of class */
+.content-background {
+  background-color: @lighter-brown;
+  border-bottom-right-radius: 25px;
+  border-top-left-radius: 25px;
+  border: 1px solid black;
+  opacity: 0.9;
+  filter: alpha(opacity=60);
```

Comment:
Not sure which element you are saying needs prefix here? Or are you referencing the box sizing inherit?